src/Schema/Events/FilterAndSort.ts

```diff
@@ -37,7 +37,26 @@ export interface CommercialFilterParams {
 }
 
 /**
- * A user applies filters to a filterable/sortable module
+ * Master list of filter params available in web and iOS, specific to auction results
+ *
+ * Intended action by a user that triggered an event
+ * @packageDocumentation
+ */
+export interface AuctionResultsFilterParams {
```

Comment:
this list may not be exhaustive, but I generated the filter params based on inspecting segment tracks when filtering auction results on web. we can collaborate on which ones need to be added, if any. They look a bit different from the CommercialFilter params.